### PR TITLE
Added GraphDatabase support via Cypher

### PR DESCRIPTION
**Pull request by jexp**

### CODE REVIEW

**Review by jerryjliu — COMMENTED** [review]
thanks! could we rename this loader to `Neo4jGraphReader`? If i'm understanding correctly this is specific to the neo4j graph db right?

**Comment by jexp** [comment]
Sure, can do that.

**Comment by jerryjliu** [comment]
I see. Can we call this `GraphDBCypherReader` then?

**Comment by jexp** [comment]
Yep that sounds great. Good idea

**Comment by jerryjliu** [comment]
@jexp thanks - also btw to resolve failing tests you can just merge main (we fixed CI)

**Comment by jexp** [comment]
Renamed + Rebased on main. But seems CI is still failing on unrelated issues?

Will provide demos for both PR's early next week, Is a Jupyter notebook w/ short recording enough? Can also do a quick blog post on it.

File: loader_hub/library.json [hunk 1]
```diff
@@ -121,6 +121,15 @@
       "apollo"
     ]
   },
+  "GraphReader": {
+    "id": "graph",
```
Comment (jerryjliu) [inline loader_hub/library.json:125]: ah you have to name this the same as the directory name "graphdb_cypher"

**Review by jerryjliu — COMMENTED** [review]
> Renamed + Rebased on main. But seems CI is still failing on unrelated issues?
> 
> Will provide demos for both PR's early next week, Is a Jupyter notebook w/ short recording enough? Can also do a quick blog post on it.

Yeah notebook + short recording is great! Totally up to you as to how you'd like to publicize

File: loader_hub/library.json (hunk 1 above)
Comment (jexp) [reply loader_hub/library.json]: Done!

**Review by jerryjliu — APPROVED** [review]

**Comment by austinmw** [comment]
Are there any examples for LlamaIndex + Neptune?

**Pull request merged**